client/my-sites/stats/hooks/use-notice-visibility-query.ts

```diff
@@ -6,6 +6,7 @@ const DEFAULT_SERVER_NOTICES_VISIBILITY = {
 	traffic_page_highlights_module_settings: false,
 	traffic_page_settings: false,
 	do_you_love_jetpack_stats: false,
+	commercial_site_upgrade: false,
```

Comment:
It'll need an easy server side work - I'll do it 😃

Reply:
Ok awesome, thank you 🎉 🙏

Reply:
Hi @annacmc, I've drafted a diff `D120306-code`. Feel free to commandeer it  and shepherd it to completion 😄

Reply:
awesome, thanks @kangzj - looks great to me! going to deploy that diff now and get this wrapped up 😄